corehq/apps/api/resources/v0_1.py

```diff
@@ -270,7 +270,7 @@ class Meta(UserResource.Meta):
 
     def obj_get_list(self, bundle, **kwargs):
         domain = kwargs['domain']
-        username = bundle.request.GET.get('username')
+        username = bundle.request.GET.get('web_username')
```

Comment:
Why does this need to change in this PR? Is it necessary for other things that are happening in this PR or is it fixing an unrelated bug? If unrelated then it might make sense to do in a separate PR so it can be easily rolled back if it turns out to be a problem.

I would not expect this to affect basic or digest auth since the credentials for those are not in GET parameters.